### PR TITLE
add precourse

### PR DESCRIPTION
**Pull request by crispine**

### CODE REVIEW

File: staff/cristina-gutierrez/precourse/calculadora/index-segunda-prueba.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
+function calculator(num1, num2) {
+    num1 = prompt("Enter the first number");
+    if (isNaN(num1)) {
+        alert("Error Nan")
+        num1 = prompt("Enter the first number")
+    }
+    let confirmationNum1 = confirm("Do you want to enter a second number?")
+    if (confirmationNum1 === false) {
```
Comment (luislourencos) [inline staff/cristina-gutierrez/precourse/calculadora/index-segunda-prueba.js:8]: You can say ' if (!confirmationNum1)' !

File: staff/cristina-gutierrez/precourse/bingo/index.js [hunk 2]
```diff
@@ -0,0 +1,60 @@
+const LINE = 5;
```
Comment (CaliZam) [inline staff/cristina-gutierrez/precourse/bingo/index.js:1]: huuuu.... ES6, remember to use "var". XD

File: staff/cristina-gutierrez/precourse/calculadora/index-segunda-prueba.js (hunk 1 above)
Comment (crispine) [reply staff/cristina-gutierrez/precourse/calculadora/index-segunda-prueba.js]: changed!

File: staff/cristina-gutierrez/precourse/bingo/index.js (hunk 2 above)
Comment (crispine) [reply staff/cristina-gutierrez/precourse/bingo/index.js]: changed!

File: staff/cristina-gutierrez/precourse/calculadora/index-segunda-prueba.js (hunk 1 above)
Comment (luislourencos) [reply staff/cristina-gutierrez/precourse/calculadora/index-segunda-prueba.js]: Thanks!!

File: staff/cristina-gutierrez/precourse/bingo/index.js [hunk 3]
```diff
@@ -1,4 +1,4 @@
-const LINE = 5;
+var LINE = 5;
```
Comment (luislourencos) [inline staff/cristina-gutierrez/precourse/bingo/index.js:1]: Name of the variables in lowercase ! sorry!

Comment (crispine) [reply staff/cristina-gutierrez/precourse/bingo/index.js:1]: It's ok if I put LINE in uppercase! It is a name convention to name constants

**Pull request merged**